api/lib/domain/usecases/validate-urls-from-release.js

```diff
@@ -79,7 +81,10 @@ function findUrlsFromTutorials(tutorials, release) {
 
 async function analyzeUrls(urlList) {
   const options = {
-    headers: { 'User-Agent': 'Mozilla/5.0 (Windows NT 6.3; Win64; x64; rv:64.0) Gecko/20100101 Firefox/80.0' },
+    headers: {
+      'User-Agent': 'Mozilla/5.0 (Windows NT 6.3; Win64; x64; rv:64.0) Gecko/20100101 Firefox/80.0',
```

Comment:
```suggestion
      'User-Agent': 'Mozilla/5.0 (Windows NT 6.3; Win64; x64; rv:64.0) Gecko/20100101 Firefox/98.0',
```
**Question**: Est-ce que l'on voudrait pas la meme version que check-url pour firefox ?

Reply:
Bien vu j'ai uniformisé les 2.